node_helper.js

```diff
@@ -60,12 +60,13 @@ module.exports = NodeHelper.create({
         let date = new Date();
         date.setDate(date.getDate() - this.config.daysInPast);
         const startDate = date.toISOString().slice(0, 10);
-        date.setDate(date.getDate() + this.config.daysInPast + this.config.daysAhead);
+        date.setDate(date.getDate() + this.config.daysAhead);
```

Comment:
The previous computation was correct because `setDate` modifies the date object.

date = now // new Date
date = yesterday // - this.config.daysInPast
date = now // + this.config.daysInPast
date = in 7 days // + this.config.daysAhead

The easiest to check is if you take the examples below with your logic
config daysInPast: 1, daysAhead: 7 => startDate=2021-01-17&endDate=2021-01-24
config daysInPast: 5, daysAhead: 5 => startDate=2021-01-13&endDate=2021-01-18

The day difference that you can see is most likely because toISOString is converting the date into UTC timezone which is several hours different from what you have in north america.

The timestamps returned by the API for the game starts are also in UTC.

Reply:
Makes sense.  I reverted.  I think I'd still like to either change this, or add a second "smartRollingLoad" option.  The way 'I' would like it to work is:  Show yesterday's results and todays upcoming games, until the upcoming games start, then I'd like to see Today and Tomorrow.  Thoughts?

Reply:
@dannoh not sure I understood it correctly. I think we should handle/discuss it in a separate github issue.